### PR TITLE
DOC: update the pandas.Series.dt.is_month_start docstring

### PR DESCRIPTION
**Pull request by janelle**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [ X] PR title is "DOC: update the <your-function-or-method> docstring"
- [ X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [ X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
################# Docstring (pandas.Series.dt.is_month_start)  #################
################################################################################

Return a boolean indicating if the date is the first day of the month.

Returns
-------
is_month_start : Series of boolean

See Also
--------
is_month_end : Returns a boolean indicating if the date is the last day of the month.

Examples
--------
>>> import pandas as pd
>>> dates = pd.Series(pd.date_range("2018-02-27", periods = 3))
>>> dates
0   2018-02-27
1   2018-02-28
2   2018-03-01
dtype: datetime64[ns]
>>> dates.dt.is_month_start
0    False
1    False
2    True
dtype: bool

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	No extended summary found
```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

This function does not require an extended summary. Peers find it very clear with a short summary too.

### CODE REVIEW

File: pandas/core/indexes/datetimes.py [hunk 1]
```diff
@@ -1727,7 +1727,32 @@ def freq(self, value):
...
+        is_month_start : Series of boolean
+
+        See Also
+        --------
+        is_month_end : Returns a boolean indicating if the date is the last day of the month.
+
+        Examples
+        --------
+        >>> import pandas as pd
+        >>> dates = pd.Series(pd.date_range("2018-02-27", periods = 3))
```
Comment (arnau126) [inline pandas/core/indexes/datetimes.py]: PEP8 doesn't allow spaces around keyword parameters. i.e.: `periods=3`, instead of `periods = 3`

Comment (arnau126) [reply pandas/core/indexes/datetimes.py]: `import pandas as pd` is not necessary.

File: pandas/core/indexes/datetimes.py [hunk 2]
```diff
@@ -1736,12 +1736,12 @@ def freq(self, value):
 
         See Also
         --------
-        is_month_end : Returns a boolean indicating if the date is the last day of the month.
+        is_month_end : Returns a boolean indicating if the date is the last day
+        of the month.
```
Comment (arnau126) [inline pandas/core/indexes/datetimes.py]: Next line should be indented:
```
        is_month_end : Returns a boolean indicating if the date is the last day
            of the month.
```

**Comment by janelle** [comment]
thanks for reviewing, adjusted line lengths and writing style now.

**Comment by janelle** [comment]
@arnau126 adjusted line lengths and writing style

**Comment by CalmDownKarm** [comment]
Fixing this.

**Comment by datapythonista** [comment]
@CalmDownKarm the function has been moved here:
- https://github.com/pandas-dev/pandas/blob/master/pandas/core/arrays/datetimes.py#L945

I think the best would be to start a fresh branch from master, and copy the code from the docstring in this PR to that location.

CC: @jbrockmendel

**Comment by datapythonista** [comment]
Discontinued and superseded by #23051

**Pull request closed without merging**